datadog_checks_base/requirements.in

```diff
@@ -6,7 +6,7 @@ cryptography==3.3.1
 ddtrace==0.32.2
 enum34==1.1.6; python_version < '3.0'
 ipaddress==1.0.22; python_version < '3.0'
-kubernetes==8.0.1
+kubernetes==12.0.1
```

Comment:
we need to be careful here in case there was a breaking change, it could break the custom checks that are using the kube client

Reply:
Unfortunately, I didn’t have much choice.
Bumping the version of the `kubernetes` client library was the only way to get access to the `coordination_v1.read_namespaced_lease()` function needed to implement this feature.

Reply:
@L3n41c If you expect lots of customers to rely on the kubernetes client in their custom checks, could you add this as an upgrade note in the agent's main changelog?